lib/inc/internal/ruby/api.hpp

```diff
@@ -21,6 +21,8 @@ namespace facter { namespace facts {
 
 namespace facter {  namespace ruby {
 
+namespace lth_dl = leatherman::dynamic_library;
```

Comment:
I have been using lth_xx throughout, so for consistency's sake I will continue to use it for now (I'll remove it from headers though). If we really want to change it to lm_xx, that is a task for another time, because it should be done everywhere.